### ISSUE TITLE
Nedá sa vycentrovať [npm package]

### ISSUE THREAD

**Issue by thefoxie** [slovensko-digital/navody-frontend#104]
Snažím sa použiť npm balík a importovať scss, tak ako odporúčajú v gov.uk

```scss
$govuk-assets-path: '/assets/';
@import "node_modules/navody-digital-frontend/all.scss";
```

avšak, url v takomto tvare zabraňujú jeho použitiu
```scss
$icon-pencil: url("../../../assets/images/icon-pencil.svg");
```

Lebo použitie .. je velmi nedeterministické
```red 
/home/foxie/Dokumenty/Code/SD/assets/images/icon-check.svg: ENOENT: no such file or directory, open '/home/foxie/Dokumenty/Code/SD/assets/images/icon-check.svg'
```

**Comment by eronisko** [comment]
Ahoj @thefoxie. Prestali sme pouzivat `govuk-image-url` helpre, lebo sa mi nepodarilo rozbehat dynamicke urls v railsovskej asset pipeline. Pouzivame namiesto toho [resolve-url-loader](https://www.npmjs.com/package/resolve-url-loader) vo webpackeri: https://github.com/slovensko-digital/navody.digital/blob/4624599ffddc9c8e3204e75bbe96ed6cf068a00a/config/webpack/environment.js. Nie je to idealny stav, ale zatial je nasa rails appka jedinym konzumentom.

Inak, s tento package sa zatial udpatuje celkom divoko, planujes s nim robit nejaku apku?

**Comment by thefoxie** [comment]
@eronisko dokoncujem lepsiesluzby.sk, ktore sa zacali robit na hackatone a chalani to vlozili dovnutra ako .njk, takze ak sa chcem dostat k zvysku (njk, ktore to pouziva), tak ho musim pouzit (ten balik). Uz mi to funguje, len som tym zabil dost casu. Potom repo s tymto projektom pojde priamo pod SD?

**Comment by eronisko** [comment]
@thefoxie repo s projektom lepsiesluzby?

**Comment by thefoxie** [comment]
> @thefoxie repo s projektom lepsiesluzby?

@eronisko ano. Ale to je asi skor otazka na PM. Ja len, ze ci by sa dal presunut ten projekt pod repozitare SD, aby to bolo public a bolo jasne o co ide.
